### DISCUSSION TITLE
pipe-magic/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#46] [Announcements]
# pipe-magic/

Copper: It’s A Thing (Sort of)

https://www.supergoodcode.com/pipe-magic/

**Comment by lucmann** [comment]
Nice blog! It's informative to me.  Out of curiosity, I'm trying hard to figure out why the name "Copper" is chosen (and why later renamed as "Kopper"), as we know, Gallium, as well as Zink,  is a kind of metal. But are they just picked up randomly as a codename thing or is there tacit logic among them that I don't realize as a non-native english speaker?

**Reply by zmike** [reply]
I think it's roughly this

![image](https://github.com/zmike/zmike.github.io/assets/234706/7edce81e-307d-49f1-a57d-d18e3f4922f5)
